src/main/java/ro/unibuc/hello/controllers/CheckUserRightsController.java

```diff
@@ -0,0 +1,24 @@
+package ro.unibuc.hello.controllers;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import ro.unibuc.hello.entities.Action;
+import ro.unibuc.hello.exceptions.UserNotAuthorizedException;
+import ro.unibuc.hello.services.CheckUserRightsService;
+
+@Controller
+public class CheckUserRightsController {
+
+    @Autowired
+    private CheckUserRightsService checkUserRightsService;
+
+    @PostMapping("/authorize/{id}")
+    @ResponseBody
+    public ResponseEntity<?> checkUserRights(@PathVariable String id, @RequestBody String actionCode) {
```

Comment:
RequestBody is generally used for a PUT/POST request, I think it would be more suitable if you used a RequestParameter or PathVariable for the actionCode.